narciuch/assignment4/test/IslandFinderTest.java

```diff
@@ -0,0 +1,33 @@
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertTrue;
+
+public class IslandFinderTest {
+  IslandFinder islandFinder = new IslandFinder();
+
+  @Test
+  public void findsMultipleIslands() {
+    boolean[][] land = {{false, true, false, true}, {true, true, false, false},
+      {false, false, true, false}, {false, false, true, false}};
+    assertTrue(islandFinder.getIslands(4, 4, land) == 3);
```

Comment:
I think it is better to use `assertEquals` from `org.junit.Assert` class. This will give you better error messages in case of test failure ("not true that bad_value is equal to expected_value" or sth similar).

Reply:
Agreed.